assets/_scss/components/_footer.scss

```diff
@@ -0,0 +1,298 @@
+.usa-footer {
+
+  button {
+    width: 100%;
+  
+    @media (min-width: $tablet-up) {
+      width: auto;
+    }
+  }
+
+  .usa-unstyled-list {
+    display: block;
+  }
+
+  .usa-footer-primary-link {
+    border-top: 1px solid $color-base-link;  
+    color: $color-base-link;
+    display: block;
+    font-weight: $font-bold;
+    margin-top: 0;
+    padding: {
+      bottom: 2rem;
+      top: 2rem;
+    }
+    text-transform: uppercase;
+
+    @media (min-width: $tablet-up) {
+      border-top: none;
+      font-size: 1.9rem;
+      text-transform: none;
+    }  
+  }  
+}
+
+.usa-footer a {
+  font-weight: normal;
+}
+
+.usa-footer-return-to-top {
+  padding: {
+    bottom: 2rem;    
+    top: 2rem;
+  }
+}
+
+.usa-footer-primary-section {
+  background-color: $color-grey-100;
+
+  a, 
+  p, 
```

Comment:
While I agree with @msecret that we should probably be a bit more explicit here, it sounds like this is something that should be more thought out as we build out the grid styles. For now, let's leave as is.

@maya would you open for re-assessing the way the grid integrates into the footer and refs this comment? Thanks!